docs/core-concepts/run.mdx

```diff
@@ -57,21 +116,49 @@ def local_udf():
 fused.run(local_udf)
 ```
 
-**Load from GitHub.** Loading the UDF requires read access to the repo.
+### Github URL
+
+{/* Need to emphasise why this isn't recommended  */}
+_When to use: [Not recommended] This is useful if you're working on a branch that you control over. This method always points to the last commit on a branch so your UDF can break without you knowing if someone else pushes a new commit or merges & deletes your branch_
 
 ```python showLineNumbers
 gh_udf = fused.load("https://github.com/fusedio/udfs/tree/main/public/REM_with_HyRiver/")
 fused.run(gh_udf)
 ```
 
-**Load from your account.** Specify the name of the UDF in your account.
+:::warning
+    We do NOT recommend you use this approach as your UDF might break if changes are done to it
+    
+    Especially using a URL pointing to a `main` branch means that your UDF will change if someone else pushes towards it, in a way that isn't visible to you.
+
+    For that reason we recommend using [git commit hash](/core-concepts/run/#git-commit-hash-recommended-for-teams) instead
+:::
+
+### Git commit hash (recommended for teams)
```

Comment:
Why is this only recommended for "teams"?

Reply:
That's a good point

Let's see it's especially recommended for teams
As if you're working by yourself you might be more aware of what changes / might break
Teams work on shared code with less awareness of the whole picture

If you raise this question it's that this isn't explained well enough, I'll do another pass
Thanks for raising

Reply:
Edited this to be for everyone, and more general